lib/MusicBrainz/Server/Report/RecordingsNoVALink.pm

```diff
@@ -0,0 +1,50 @@
+package MusicBrainz::Server::Report::RecordingsNoVALink;
+use Moose;
+use MusicBrainz::Server::Constants qw( $VARTIST_ID );
+
+with 'MusicBrainz::Server::Report::RecordingReport';
+
+sub query {
+	# Last two artists are actually named "Various Artists"
+    "
+        SELECT
+            r.id AS recording_id,
+            row_number() OVER (ORDER BY r.artist_credit, r.name)
+        FROM recording r
+        JOIN artist_credit_name acn on acn.artist_credit = r.artist_credit
+        WHERE acn.name = 'Various Artists'
+          AND acn.artist != $VARTIST_ID
+          AND acn.artist != 242879
+          AND acn.artist != 344239
+          AND acn.artist != 981120
```

Comment:
Right, I would join with the artist table here and check `a.name != 'Various Artists'` instead.